backoffice/backoffice/common/views.py

```diff
@@ -38,3 +51,67 @@ def create(self, request, *args, **kwargs):
         serializer.is_valid(raise_exception=True)
         serializer.save()
         return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+
+class BaseWorkflowViewSet(viewsets.ModelViewSet):
+    """
+    Base for any “Workflow” ModelViewSet.
+    Subclasses must set:
+      • schema_name
+    """
```

Comment:
I would add a quick explanation what that means. i.e. the name should match with the inspire-schemas etc